.changeset/famous-buckets-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-dropdown": patch
+---
+
+[MultiSelect] Remove `showOpenerLabelAsText` from sharedProps that are passed to SelectOpener
```

Comment:
minor: it will also be updated in SingleSelect!

```suggestion
[MultiSelect and SingleSelect] Remove `showOpenerLabelAsText` from sharedProps that are passed to SelectOpener
```